src/main/java/friday/Ui.java

```diff
@@ -0,0 +1,95 @@
+package friday;
+public class Ui {
+
```

Comment:
Should these public methods have their own Javadocs to explain the usage of the methods. I am also seeing this issue in some places within your code.